packages/platform/setup-accounts.js

```diff
@@ -56,16 +67,15 @@ export default ({ mergeUserCartsOnLogin = true } = {}) => {
         { address: newUser.services.facebook.email, verified: true },
       ];
     }
-    if (!guest && !skipEmailVerification) {
-      Meteor.setTimeout(() => {
-        const { sendVerificationEmail = true } = Accounts._options; // eslint-disable-line
-        Accounts.sendVerificationEmail(user._id);
-      }, 1000);
+    if (!newUser.guest && !options?.skipEmailVerification) {
+      accountsPassword.sendVerificationEmail(
+        Users.findOne({ _id: user._id }).primaryEmail()?.address,
+      );
     }
     return newUser;
   });
 
-  Accounts.removeOldGuests = (before) => {
+  accountsServer.removeOldGuests = (before) => {
```

Comment:
I think you can either remove this for now or add an admin mutation that allows calling that method.